contracts/FountainV1.sol

```diff
@@ -592,7 +595,8 @@ contract FountainV1 {
         moneyPool.duration = currentMoneyPool.duration;
         moneyPool.want = currentMoneyPool.want;
         moneyPool.exists = true;
-        moneyPool.previousMoneyPoolId = moneyPoolCount;
```

Comment:
@tauren this here seems off also. moneyPoolCount here is the id of the newly created MP, it doesn't make sense that previousMoneyPoolId is being set to it.

Reply:
Very true. This should be assigned to `moneyPoolId` like you've done here. But we don't need the `moneyPool.id` assignment.